content/en/blog/_posts/2019-03-29-kube-proxy-subtleties-debugging-an-intermittent-connection-resets.md

```diff
@@ -139,15 +139,20 @@ Once we understand the root cause, the fix is not hard. There are at least 2
 ways to address it.
 
 - Make conntrack more liberal on packets, and don’t mark the packets as
-  *INVALID*. In Linux, you can do this by `echo 1 >
-  /proc/sys/net/ipv4/netfilter/ip_conntrack_tcp_be_liberal`.
+  _INVALID_. In Linux, you can do this by `echo 1 > /proc/sys/net/ipv4/netfilter/ip_conntrack_tcp_be_liberal`.
 - Specifically add an iptables rule to drop the packets that are marked as
-  *INVALID*, so it won’t reach to client pod and cause harm.
+  _INVALID_, so it won’t reach to client pod and cause harm.
 
 The [fix](https://github.com/kubernetes/kubernetes/pull/74840) is available in v1.15+.
 However, for the users that are affected by this bug, there is a way to mitigate the
 problem by applying the following rule in your cluster.
 
+_Update_: With worker nodes on kernel version 4.14 and above, the path mentioned in the script to set the sysctl might be different.
+The recommendation is to connect to the worker node through SSH and run the following the command.
+`echo 1 > /proc/sys/net/ipv4/netfilter/ip_conntrack_tcp_be_liberal`.
+If the command returns `No such file or directory`, that means the netfilter directory likely uses the sysctl path`/proc/sys/net/netfilter/nf_conntrack_tcp_be_liberal`.
+You must update the command in the DaemonSet below to use the correct path.
```

Comment:
This seems to be the only / key change. @taniaduggal are you using an editor that automatically reformats Markdown?

We'd prefer to only see the relevant changes. To do that, you could copy this section and redo your PR from the _main_ branch. When you come to commit your work, use `git add -p` or the equivalent IDE tool to let you select which changes to commit.